docker/Dockerfile

```diff
@@ -18,6 +18,10 @@ ENV LD_LIBRARY_PATH /opt/nvidia/hpc_sdk/Linux_x86_64/24.1/comm_libs/nvshmem/lib:
 ENV LD_LIBRARY_PATH /opt/nvidia/hpc_sdk/Linux_x86_64/24.1/math_libs/lib64:$LD_LIBRARY_PATH
 ENV CUDA_HOME /opt/nvidia/hpc_sdk/Linux_x86_64/24.1/cuda
 
+# some useful things
```

Comment:
For final merge, add these to the`apt install` command further up the in the `Dockerfile`.

Reply:
fixed.